features/discovery/src/test/java/org/opennms/netmgt/discovery/messages/DiscoveryJobTest.java

```diff
@@ -1,116 +0,0 @@
-
-    @Test
-    public void testCalculateTaskTimeout() throws Exception {
-        final List<IPPollRange> m_ranges = new ArrayList<IPPollRange>();
-        // Make 3 ranges of 5 addresses each
-        for (int i = 0; i < 3; i++) {
-            IPPollRange ipPollRange = new IPPollRange(null, null, "127.0.1." + ((i * 5) + 1), "127.0.1." + ((i * 5) + 5), 50, 2);
-            m_ranges.add(ipPollRange);
-        }
-        DiscoveryJob discoveryJob = new DiscoveryJob(m_ranges, "Bogus FS", "Bogus Location", Double.MAX_VALUE);
-        // Each task is taking 750 ms so totalTaskTimeout = 750 ms * 3 (number of tasks) = 2250 ms
-        assertEquals(Math.round(2250 * 1.5), discoveryJob.calculateTaskTimeout()); 
-
-        // Set the rate to 1 per second and make sure that the timeout is extended
-        // by 1000 milliseconds per all 15 IP addresses
-        discoveryJob = new DiscoveryJob(m_ranges, "Bogus FS", "Bogus Location", 1.0);
-        assertEquals(Math.round(2250 * 1.5) + (15 * 1000), discoveryJob.calculateTaskTimeout()); 
-    }
-
-    @Test
-    public void testTimeoutIntegerMaxValueMinus1() throws Exception {
-        List<IPPollRange> m_ranges = new ArrayList<IPPollRange>();
-
-        // Minimum value that should be smaller than Integer.MAX_VALUE:
-        //
-        // (Integer.MAX_VALUE - 1) * timeout * retries * fudge_factor / fudge_factor
-        //       (2^31 - 1)        *    1    *    1    *     1.5      /     1.5
-        //  ((2^31 - 1) / 3 * 2)   *    1    *    1    *     1.5
-
-        // Start the range at 0.0.0.1 so that we don't count zero as a value
-        IPPollRange ipPollRange = new IPPollRange(null, null, "0.0.0.1", "85.85.85.84", 1, 0);
-        m_ranges.add(ipPollRange);
-        DiscoveryJob discoveryJob = new DiscoveryJob(m_ranges, "Bogus FS", "Bogus Location", Double.MAX_VALUE);
-        assertEquals(Integer.MAX_VALUE - 1, discoveryJob.calculateTaskTimeout());
-    }
-
-    @Test
-    public void testTimeoutIntegerMaxValue() throws Exception {
-        List<IPPollRange> m_ranges = new ArrayList<IPPollRange>();
-
-        // Start the range at 0.0.0.1 so that we don't count zero as a value
-        IPPollRange ipPollRange = new IPPollRange(null, null, "0.0.0.1", "85.85.85.85", 1, 0);
-        m_ranges.add(ipPollRange);
-        DiscoveryJob discoveryJob = new DiscoveryJob(m_ranges, "Bogus FS", "Bogus Location", Double.MAX_VALUE);
-        assertEquals(Integer.MAX_VALUE, discoveryJob.calculateTaskTimeout());
-    } 
-
-    @Test
-    public void testTimeoutLargerThanIntegerMaxValue1() throws Exception {
-        List<IPPollRange> m_ranges = new ArrayList<IPPollRange>();
-        IPPollRange ipPollRange = new IPPollRange(null, null, "0.0.0.0", "89.0.0.0", 1, 0);
-        m_ranges.add(ipPollRange);
-        DiscoveryJob discoveryJob = new DiscoveryJob(m_ranges, "Bogus FS", "Bogus Location", Double.MAX_VALUE);
-        assertEquals(Integer.MAX_VALUE, discoveryJob.calculateTaskTimeout());
-    } 
-
-    @Test
```

Comment:
The other tests just throw some values other than 1 into the timeout/retries values to make sure that the BigInteger multiplication is correct.